beacon-chain/state/v3/getters_block.go

```diff
@@ -0,0 +1,99 @@
+package v3
```

Comment:
This is the **only** difference to the v2 file, we should do something about this, it's unmaintainable